flow/table.go

```diff
@@ -43,6 +52,21 @@ type Handler struct {
 	callback ExpireUpdateFunc
 	every    time.Duration
 }
+type EBPFFlow struct {
```

Comment:
can you move all the ebpf related functions into a dedicated file, it will be easier to read

Reply:
done